rust/src/transformations/make_stable_expr/namespace_dt/expr_datetime_component/pseudocode/match_datetime_component.py

```diff
@@ -0,0 +1,23 @@
+# type: ignore
+def match_datetime_component(
+    temporal_function: TemporalFunction,
+) -> DataType | None:
+    ({
```

Comment:
```suggestion
    return ({
```
If you want to keep it python-y.